parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -83,6 +83,9 @@ if [[ -n "${OUTBOUND_COMMAND}" ]]; then
         eval $PROXY_VARS
     fi
     retrycmd_if_failure 50 1 5 $OUTBOUND_COMMAND >> /var/log/azure/cluster-provision-cse-output.log 2>&1 || exit $ERR_OUTBOUND_CONN_FAIL;
+else
```

Comment:
just curious, where do we have logic to skip outbound-check according to this file?

Reply:
not in agentbaker